subsys/bluetooth/mesh/adv_ext.c

```diff
@@ -208,6 +210,29 @@ static void send_pending_adv(struct k_work *work)
 	int err;
 
 	adv = CONTAINER_OF(work, struct bt_mesh_ext_adv, work.work);
+
+	if (atomic_test_and_clear_bit(adv->flags, ADV_FLAG_SENDED)) {
```

Comment:
@trond-snekvik Please note that this is a new bug fix that I put in the same PR as it is associated with the first commit.